### PR TITLE
Reduce repetitive method calls via local variables

### PR DESCRIPTION
**Pull request by ashmaroli**
just some syntactic sugar..

### CODE REVIEW

File: lib/kramdown/parser/kramdown/link.rb [hunk 1]
```diff
@@ -92,8 +92,9 @@ def parse_link
           emit_warning = !@src[1]
           link_id = normalize_link_id(@src[1] || alt_text)
           if @link_defs.key?(link_id)
-            add_link(el, @link_defs[link_id][0], @link_defs[link_id][1], alt_text,
-                     @link_defs[link_id][2] && @link_defs[link_id][2].options[:ial])
+            link_def_id = @link_defs[link_id]
```
Comment (gettalong) [inline lib/kramdown/parser/kramdown/link.rb]: That would be a `link_def`, not a `link_def_id`.

Comment (ashmaroli) [reply lib/kramdown/parser/kramdown/link.rb]: Agreed. :slightly_smiling_face:

**Comment by ashmaroli** [comment]
*Reminder: Please use `Squash and Merge` option to merge this PR via GitHub interface*

**Comment by gettalong** [comment]
Thanks for the changes!

**Pull request closed without merging**